<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidget_Halcon_LoadImageClass</class>
 <widget class="QMainWindow" name="QWidget_Halcon_LoadImageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1030</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWidget_Halcon_LoadImage</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>491</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>生产统计</string>
     </attribute>
     <widget class="QTableView" name="table_static1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="table_static2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>481</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>PLC I/O</string>
     </attribute>
     <widget class="QTableView" name="table_IO">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>相机1参数</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>相机2参数</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>相机3参数</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1491</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>492</width>
          <height>430</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>410</width>
          <height>430</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>492</width>
          <height>430</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>410</width>
          <height>430</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>492</width>
          <height>430</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>410</width>
          <height>430</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>501</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>报警</string>
       </property>
       <widget class="QTableView" name="table_alarm">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>491</width>
          <height>221</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>运行</string>
       </property>
       <widget class="QTextEdit" name="textEdit_status">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>491</width>
          <height>231</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>520</y>
      <width>501</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>125</height>
        </size>
       </property>
       <property name="title">
        <string>SystemStatus</string>
       </property>
       <widget class="QPushButton" name="Indicator_Comm">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>23</y>
          <width>86</width>
          <height>23</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Communication</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Indicator_Camera">
        <property name="geometry">
         <rect>
          <x>105</x>
          <y>23</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Indicator_Camera2">
        <property name="geometry">
         <rect>
          <x>186</x>
          <y>23</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Indicator_Camera3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Indicator_Database">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>52</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>DataBase</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>125</height>
        </size>
       </property>
       <property name="title">
        <string>Parameters</string>
       </property>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>69</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Recipe1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Recipe2</string>
         </property>
        </item>
       </widget>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera1</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera2</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Camera3</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>125</height>
        </size>
       </property>
       <property name="title">
        <string>Buttons</string>
       </property>
       <widget class="QPushButton" name="btn_loadImage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>触发</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_loadcontinuImage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>连续触发</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>171</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sendButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exitButton">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_saveImage">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>保存图片</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_importData">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>90</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>导出数据</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>camera1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>camera2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>camera3</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionCLose"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionNormalSize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionCLose">
   <property name="text">
    <string>CLose</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionNormalSize">
   <property name="text">
    <string>NormalSize</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QWidget_Halcon_LoadImage.qrc"/>
 </resources>
 <connections/>
</ui>
